ooDesign/frankensteinWithConstructors/solutionM/src/main/java/oointro/Frankenstein.java

```diff
@@ -0,0 +1,77 @@
+package oointro;
+
+public class Frankenstein {
+    private String cString;     // combination string of song and pet info
+    private double alphaAvg;    // average number of alphanumerics
+    private Song frankenSong;   // aggregate class
+    private Pet frankenPet;     // aggregate class
+
+
+    // Constructor Methods
+    public Frankenstein() {
+        this(new Song(), new Pet()); // this uses last constructor with 2 parameters to reduce duplicate code
+    }
+

```

Comment:
modify the other constructor calls to use this() as well please.   That is a better practice when using constructors as it means that the actual instance variable is only touched in one constructor.   So this(song, new Pet()) and this(new Song(), pet)...